spaceship/lib/spaceship/portal/portal_client.rb

```diff
@@ -465,6 +465,22 @@ def provisioning_profiles(mac: false)
       end
     end
 
+    ##
+    # this endpoint is used by Xcode to fetch provisioning profiles.
+    # The response is an xml plist but has the added benefit of containing the appId of each provisioning profile.
```

Comment:
Add something to the comment to the effect of "Using this method is more efficient than `provisioning_profiles` because no secondary api calls are necessary" or comment on `provisioning_profiles` talking about it being less efficient.